Sources/HaCWebsiteLib/Events/EventManager.swift

```diff
@@ -0,0 +1,44 @@
+import Foundation
+
+public struct EventManager {
```

Comment:
Do we need an event manager?

Ideally we keep this website as stateless as possible, just something to interact with other stateful sources of truth (Database, workshop repo, etc...). In the case where there are expensive operations (such as pulling a Git repo), some layer of caching may be called for as an optimisation (ie `WorkshopManager`). In the case of _events_, we'll be pulling them from our database, which is not an expensive operation.

Do you think it would be less complex if we had an `events` computed property somewhere that does the logic you have in `update()` on every call?